tests/test_math_stat.py

```diff
@@ -0,0 +1,96 @@
+import aesara.tensor as at
+import numpy as np
+import pytest
+from aesara.tensor.random.basic import NormalRV
+
+from aeppl.rewriting import construct_ir_fgraph
+
+
+@pytest.mark.parametrize(
+    "mu_x, mu_y,  x_shape, y_shape, new_rv_mu",
+    [
+        (
+            np.array([1, 10, 100]),
+            np.array(2),
+            None,
+            None,
+            np.array([3, 12, 102]),
+        ),
+        (np.array([1, 10, 100]), np.array(2), (), (), np.array([3, 12, 102])),
+        (
+            np.array([1, 10, 100]),
+            np.array(2),
+            (5, 3),
+            (),
+            np.broadcast_to(np.array([3, 12, 102]), (5, 3)),
+        ),
+        (
+            np.array([[1, 10, 100]]),
+            np.array([[0.2], [2], [20], [200], [2000]]),
+            None,
+            None,
+            np.array(
+                [
+                    [1.2, 10.2, 100.2],
+                    [3, 12, 102],
+                    [21, 30, 120],
+                    [201, 210, 300],
+                    [2001, 2010, 2100],
+                ]
+            ),
+        ),
+        (
+            np.broadcast_to(np.array([1, 10, 100]), (5, 3)),
+            np.array([2, 20, 200]),
+            (2, 5, 3),
+            None,
+            np.broadcast_to(np.array([3, 30, 300]), (2, 5, 3)),
+        ),
+        (
+            np.array([[1, 10, 100]]),
+            np.array([[0.2], [2], [20], [200], [2000]]),
+            (2, 5, 3),
+            None,
+            np.broadcast_to(
+                np.array(
+                    [
+                        [1.2, 10.2, 100.2],
+                        [3, 12, 102],
+                        [21, 30, 120],
+                        [201, 210, 300],
+                        [2001, 2010, 2100],
+                    ]
+                ),
+                (2, 5, 3),
+            ),
+        ),
+        (
+            np.array(1),
+            np.array(6),
+            (5, 1),
+            (1,),
+            np.full((5, 3), 7),
+        ),
+    ],
+)
+def test_add_independent_normals(mu_x, mu_y, x_shape, y_shape, new_rv_mu):
```

Comment:
Would it make sense to compute `new_rv_mu` from `mu_x`, `mu_y`, `x_shape` and `y_shape` directly?

Reply:
Ah, just via NumPy manipulations? I think that works